modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/internal/object/system/test/AccountEntrySystemObjectDefinitionManagerTest.java

```diff
@@ -52,6 +59,75 @@ public void tearDown() throws Exception {
 		super.tearDown();
 	}
 
+	@Test
+	public void testAddBaseModel() throws Exception {
+
+		// With permission
+
+		User user1 = TestPropsValues.getUser();
+
+		setUser(user1);
+
+		String description1 = RandomTestUtil.randomString();
```

Comment:
Can you rename it to `description` and `name` and  reuse these variables to avoid having xxx1, xxx2?

You probably will need to move the block below up, right after `_assertCount(accountEntriesCount + 1);`
```
		AccountEntry accountEntry1 = _accountEntryLocalService.getAccountEntry(
			accountEntryId1);

		Assert.assertEquals(description1, accountEntry1.getDescription());
		Assert.assertEquals(name1, accountEntry1.getName());
```